<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <node name="robot_to_mc" pkg="robot_to_mc_node" type="robot_to_mc_node" output="screen">
    <param name="frequency" value="100.0" />
    <param name="dst_addr"  value="127.0.0.1" />
    <param name="dst_port" value="5432" />
    <param name="src_port" value="5432" />
  </node>


</launch>
